Sources/Vapor/Validation/ValidationsError.swift

```
@@ -37,3 +37,36 @@ extension ValidationsError: AbortError {
         self.description
     }
 }
+
+extension ValidationsError: CustomErrorResponseConvertible {
+    internal struct ErrorResponse: Codable {
+        var error: Bool
+        var reason: String
+        var validationErrors: [String: String]
+    }
+    
+    public func customResponse() -> Response {
+        // collect validationErrors
+        var validationErrors: [String: String] = [:]
+        
+        self.failures.forEach { failure in
+            validationErrors = validationErrors.merging([failure.key.description: failure.failureDescription ?? ""]) {
+                return $0 + ", " + $1
+            }
+        }
+        
+        // create a Response with appropriate status
+        let response = Response(status: status, headers: headers)
+        
+        // attempt to serialize the error to json
+        do {
+            let errorResponse = ErrorResponse(error: true, reason: reason, validationErrors: validationErrors)
+            response.body = try .init(data: JSONEncoder().encode(errorResponse))
+            response.headers.replaceOrAdd(name: .contentType, value: "application/json; charset=utf-8")
+        } catch {
+            response.body = .init(string: "Oops: \(error)")
```

We should just return the error here and let ErrorMiddleware figure out how it wants to create the failsafe response without repeating it in each custom error. `ResponseEncodable` already returns a future which supports errors.